etsc.config.js

```diff
@@ -0,0 +1,16 @@
+
+module.exports = {
+    // Supports all esbuild.build options
+    tsConfigFile: './tsconfig.node.json',
+    esbuild: {
+        minify: false,
+        target: "es2020",
+        outdir: './dist/server',
+        bundle: true,
+        loader: { ".node": "file" },
+        external: [
+            "esbuild",
+            "lightningcss"
+        ]
+    },
+};
```

Comment:
The idea is the following:
there is two ways how we can start the server in production env (docker)
1. either include the `node_modules` in the docker image so we can use `ts-node` to start the server, 
2. Build the server app and bundle it so we did not need the `node_modules`
 I did not find a good way to build the server with `Vite`, so I use `esbuild` which is also used with `Vite` in dev mode